azure-migrate/dependencies-at-scale/README.md

```diff
@@ -0,0 +1,100 @@
+# Dependency analysis using Powershell in Azure Migrate
+
+This powershell module helps you automate enabling and disabling dependency analysis on a large number of VMware VMs discovered in Azure Migrate. These are applicable to the agentless dependency analysis feature. 
+
+## Overview 
+With this PowerShell module you can:
+- Get a list of discovered VMware machines along with the status on whether dependency analysis is currently enabled.
+- Enable or disable dependency analysis on a large number of VMs.
+- Visualize network connections using downloaded dependency data. The downloaded dependency data can be imported into a PowerBI to visualize a map of network connections. The data can be downloaded for one appliance at a time. You can filter the map by servers and processes
+
+## Current Limitations:
+- Agentless dependency analysis can be enabled on 1000 VMs concurrently.
+
+## Pre-requisites
+Before you get started, you need to do the following:
+- [Review](https://docs.microsoft.com/azure/migrate/migrate-support-matrix-vmware#dependency-analysis-requirements-agentless) supported operating systems, and required permissions for agentless dependency analysis.
+- Have an Azure Migrate project and the Azure Migrate:Server Assessment tool is added to the project.
+- Set up an [Azure Migrate appliance](https://docs.microsoft.com/azure/migrate/migrate-appliance) to discover the on-premises VMware VMs. The appliance discovers on-premises machines, and sends metadata and performance data to Azure Migrate:Server Assessment.
+- Ensure that VMware Tools (later than 10.2) is installed on each VM you want to analyze.
+- Ensure you have downloaded the AzMig_Dependencies powershell module in this folder.
+- Ensure you have PowerShell available. It is recommended you use the [latest version of PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell).
+- Ensure you have the [Azure PowerShell](https://docs.microsoft.com/powershell/azure/install-az-ps) module installed.
+- The Az.Accounts module is required and must be at least version 2.2.0 or higher.
+
+## Log in to Azure
+
+1. Log into your Azure subscription using the Connect-AzAccount cmdlet.
+
+```PowerShell
+Connect-AzAccount
+```
+
+If using Azure Government, use the following command.
+
+```PowerShell
+Connect-AzAccount -EnvironmentName AzureUSGovernment
+```
+
+2. Select the subscription in which you have created the Azure Migrate project 
+
+```PowerShell
+Select-AZSubscription -subscription "Fabrikam Demo Subscription"
```

Comment:
```suggestion
Set-AzContext -Subscription "Fabrikam Demo Subscription"
```

Reply:
Use the cmdlet name instead of an alias and specify parameters in the same case they tab expand to or show up in intellisense as.